CHANGELOG.en-US.md

```diff
@@ -19,7 +19,7 @@ timeline: true
 
 `2018-12-02`
 
-3.11.0 brings two new Component, and a lot of exciting changes and new features.
+3.11.0 brings two new Components, and a lot of exciting changes and new features.
```

Comment:
I prefer to `3.11.0 brings two new Components, a lot of exciting changes and new features`.

Reply:
`3.11.0 brings two new Components and a lot of exciting changes and new features.`

Reply:
But usually one `and` is enough.
For example:
``````
I have a banana, an orange and some apples.
``````

Reply:
`3.11.0 brings two new Components, a lot of exciting changes and new features.`

Reply:
done